src/frontend/src/flows/manage.ts

```diff
@@ -328,19 +327,26 @@ const bindRemoveListener = (
     if (isOnlyDevice) {
       return alert("You can not remove your last device.");
     } else {
+      let shouldProceed;
```

Comment:
nit: this suggests `shouldProceed` is mutable whereas really it's just that we don't know how to initialize it here. Also, using the `let` there's a risk that someone will add a branch to the `if/else` and will forget to assign `shouldProceed`.

How about making sure `shouldProceed` is always set correctly?

```
const shouldProceed = sameDevice ? confirm("this be yar last device") : confirm("ya sure, pirate?");
...